app-hw-37/src/FullScreenCard/FullScreenCard.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode } from 'react';
+import { useState, useEffect } from 'react';
+import { getPost } from '../getPosts';
+import './fullScreenCard.css';
+type Post = {
+	id: number,
+	image?: string,
+	text: string,
+	date: string,
+	lesson_num?: number,
+	title: string,
+	description: string,
+	author: number
+}
+
+type Props = {
+	id: number,
+	goBack: () => void
+}
+export const FullScreenCard = (props: Props) => {
+	const [post, setPost] = useState<Post>({
```

Comment:
prefer undefined then empty object

Reply:
gives TS mistake: Argument of type 'null' is not assignable to parameter of type 'Post'